listings_app/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for listings_app project.
+
+Generated by 'django-admin startproject' using Django 2.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'h)wmbja@&kvo3#0yyv@piqcp6*(xs=r5!2grbiej8y2!issrn@'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
**DESCRIPTION** 
This setting allows us to see detailed errors while we work locally, for example if you try to load a page that is broken Django will spit out a big bunch of info to give you hints on what's wrong.

We don't ever want to leave this setting on in production (we will split the settings into Dev, Test and Production to help us manage differenced in behaviours based on an environment). Leaving this setting on in production can give attackers enough information to access the site or steal some sensitive information.